temporal/api/cloud/auditlog/v1/message.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package temporal.api.cloud.auditlog.v1;
+
+option go_package = "go.temporal.io/api/cloud/auditlog/v1;auditlog";
+option java_package = "io.temporal.api.cloud.auditlog.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::AuditLog::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.AuditLog.V1";
+
+import "google/protobuf/timestamp.proto";
+
+// LogRecord represents an audit log entry from Temporal, structured for easy parsing and analysis.
+message LogRecord {
+  // Time when the log was emitted from the source
+  google.protobuf.Timestamp emit_time = 1;
+
+  // Level of the log, i.e. info, warn, error etc
+  LogLevel level = 2;
+
+  // Operator email address
+  string user_email = 3;
+
+  // Operator IP address or server name
+  string caller_ip_address = 4;
+
+  // Operation performed
+  string operation = 5;
+
+  // Detailed information about the operation.
+  OperationDetail details = 6;
+
+  // Operation status from API call
+  string status = 7;
+
+  // Admin or System
+  LogCategory category = 8;
+
+  // Version of the log entry
+  int32 version = 9;
+
+  // Unique ID for the log record.
+  string log_id = 10;
+}
+
+// OperationDetails includes potential more detailed operation logs extracted from the different fields in the
+// Temporal API logs or third party logs, e.g. auth0
+message OperationDetail {
```

Comment:
Do we have flexibility to structure this better. 
My worry is this message will become a dumping ground of every information of the about the operation.
From the top of my head can we have a NamespaceDetails, UserDetails, AccountDetails, ServiceAccountDetails, ApikeyDetails, UserGroupDetails, etc..

Reply:
I was told we cannot change this structure at this time. If we can, I'd like to request all operation-specific things be in the operation area (including operation name and status) instead of just some